functions/usage/index.js

```diff
@@ -267,6 +267,51 @@ exports.cloudifyUsage = function cloudifyUsage(req, res) {
                     'cloudify_usage_last_login',
                     (default_value = null)
                 ),
+                cloudify_usage_max_deployments: _getValue(
+                    row_data,
+                    'cloudify_usage_max_deployments',
+                    (default_value = null)
```

Comment:
while we're at it, let's fix the obvious bug here, and not add more occurrences of it. `(default_value = null)` sets the GLOBAL variable `default_value` to null, and then the expression returns null, so that's passed as the third argument.

No reason to leak a global here.

The right syntax in js is:
```javascript
                cloudify_usage_max_deployments: _getValue(
                    row_data,
                    'cloudify_usage_max_deployments',
                    null
                ),
```

Reply:
This was changed (parentheses added) in #8 but the previous impl also was wrong